Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchable.cs

```diff
@@ -107,6 +107,14 @@ protected override void OnValidate()
             bounds.y = Mathf.Max(bounds.y, 0);
         }
 
+        void OnEnable()
+        {
+            if (touchableCollider == null)
+            {
+                SetTouchableCollider(GetComponent<BoxCollider>());
```

Comment:
there's no tooltip for touchableCollider - can you add it please and also describe the default behavior there, so ppl know that if this is not set its own gameobjects BoxCollider will be used.

Reply:
(please also add default behavior description to /// comment that will appear in docs

Reply:
>(please also add default behavior description to /// comment that will appear in docs

@CDiaz-MS  please remember to do this as well, I saw you added the tooltip

Reply:
Added doc comments to all public fields